SubQuery/Multichain/src/mappings/mappingHandlers.ts

```diff
@@ -1,14 +1,20 @@
-import { Bridge } from "../types";
+import { Bridge, State } from "../types";
 import Status from "../utils/status";
 import {
+  ChangeTransferTimeLockLog,
   ClaimLog,
+  HoldClaimLog,
   ProvisionConfirmLog,
+  ReleaseClaimLog,
   RemoveProvisionLog,
 } from "../types/abi-interfaces/BridgeAbi";
 
 import assert from "assert";
 import { CosmosEvent } from "@subql/types-cosmos";
 
+const BRIDGE_CONTRACT_ADDRESS = "0x482f8229cA58a25C824634758963dD082F36690b";
+const TRANSFER_TIME_LOCK = BigInt(1800); // 30 minutes
```

Comment:
Rather writing with const values, fetching them would be appropriate at initialization

Reply:
Fixed` bridge contract address` - fetching it from logs instead of hardcoding it
`TRANSFER_TIME_LOCK` not able to fetch from chain. Somehow Subquery node not allowing to connect with RPC node from within its mapping handler.
Getting Network error:
> Error: could not detect network (event="noNetwork", code=NETWORK_ERROR, version=providers/5.7.2)

Note:
No issue with `ethers` config, because same script fetching `TRANSFERLOCK` correctly when run separately.